File: Networking/src/main/java/response/AllParticipantsResponse.java
package response;

import example.example.Participant;

import java.util.Collection;

public class AllParticipantsResponse implements Response {
    private final Collection<Participant> participants;

    public AllParticipantsResponse(Collection<Participant> participants) {
        this.participants = participants;
    }

    public Collection<Participant> getParticipants() {
        return participants;
    }
}

File: Networking/Response/AllParticipantsResponse.cs
using System.Text.Json.Serialization;
using mpp_proiect_csharp_DianaGliga11.Model;

namespace Networking.Response
{

    public class AllParticipantsResponse : IResponse
    {
        [JsonPropertyName("participants")]
        public List<Participant> Participants{ get; set; }

        [JsonConstructor]
        public AllParticipantsResponse()
        {
        }

        public AllParticipantsResponse(List<Participant> participants)
        {
            this.Participants = participants;
        }
    }
}